src/VisualStudio/Core/Def/Implementation/UnusedReferences/ProjectAssets/ProjectAssetsReader.cs

```diff
@@ -66,7 +66,8 @@ internal static class ProjectAssetsReader
             }
 
             var autoReferences = projectAssets.Project?.Frameworks?.Values
-                .SelectMany(framework => framework.Dependencies?.Keys.Where(key => framework.Dependencies[key].AutoReferenced))
```

Comment:
It feels like usage of the original code should have resulted in a nullable type argument to `ImmutableHashSet`. Was this the case or no?